packages/compass-e2e-tests/tests/auto-connect.test.ts

```diff
@@ -287,22 +287,23 @@ describe('Automatically connecting from the command line', function () {
         currentWindow = allWindows.find((w) => w !== currentWindow) as string;
         await browser.switchToWindow(currentWindow);
         // the new window should open on the welcome tab
-        const currentActiveTab = await browser.$(
+        const currentActiveTab = browser.$(
           Selectors.workspaceTab({ active: true })
         );
         const type = await currentActiveTab.getAttribute('data-type');
         return type === 'Welcome';
       });
 
       // no toasts to signify that anything is connecting
-      const toasts = await browser.$(Selectors.LGToastContainer).$$('div');
-      expect(toasts).to.have.lengthOf(0);
+      const numToasts = await browser.$(Selectors.LGToastContainer).$$('div')
+        .length;
+      expect(numToasts).to.equal(0);
```

Comment:
The change to the result of $$ kinda breaks the nice to.have.lengthOf. Checking the length (which we have to await) is basically the same until the test fails. The error is now 1 != 0 rather than expected [a, b, c, d] to have length of 0. Which is a bit less nice but I just found this fix easiest at scale.